stm/Inc/motorControl.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
```

Comment:
dependecy diagram should look as start, i.e. have one to many relation, NO MANY TO MANY

Reply:
didnt understand this, i did this like tmc2590.h, which also didnt use #ifndef